### PR TITLE
silly 

### PR DESCRIPTION
**Pull request by Telecoz**
added some cool stuff you should add

### CODE REVIEW

**Comment by aeolus-1** [comment]
bussin pr

**Comment by Telecoz** [comment]
thats what im sayin bro ong

File: app.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def validate_username(username):
 
 # Validate the message
 def validate_message(message):
-    return bool(message) and len(message) <= 400
+    return bool(message) and len(message) <= 10000000000
```
Comment (Telecoz) [inline app.py]: ignore that got a lil silly

Comment (Phntmm) [reply app.py]:
```suggestion
    return bool(message) and len(message) <= 10000000000000000000000000000000000000000000000000000000000000000
```

Comment (Phntmm) [reply app.py]: ^ good to copy paste propoganda

**Pull request closed without merging**